src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CSharpCodeGenerationTests.cs

```diff
@@ -3516,5 +3516,59 @@ public class ClassWithGenericMethod {
                   }
                 ");
         }
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/56267", TargetFrameworkMonikers.NetFramework)]
```

Comment:
This is what is done further up in this file on another test, using ActiveIssue to point to a closed GitHub issue.

Another option would potentially be `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/runtime/issues/56267")]`.

Reply:
We have examples where there is reason description with link, such as:

https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/MinLengthAttributeTests.cs#L75